pinot-controller/src/main/java/org/apache/pinot/controller/validation/RealtimeSegmentValidationManager.java

```diff
@@ -100,6 +100,7 @@ protected void processTable(String tableNameWithType, Context context) {
           IngestionConfigUtils.getStreamConfigMap(tableConfig));
       if (streamConfig.hasLowLevelConsumerType()) {
         _llcRealtimeSegmentManager.ensureAllPartitionsConsuming(tableConfig, streamConfig);
+        _llcRealtimeSegmentManager.uploadToSegmentStoreIfMissing(tableConfig);
```

Comment:
We might want to make it configurable in case users want to run pinot without deep storage

Reply:
+1

Reply:
Added a controller periodic job config for this: controller.realtime.segment.validation.uploadToSegmentStoreIfMissing